src/main/java/frc/robot/commands/vision/TestVisionDistance.java

```diff
@@ -60,7 +71,6 @@ public void execute(){
       // reset the timer
       m_endTimer.reset();
       // If kPrintDelay seconds have passed, print the data
-      // TODO: Consider using shuffleboard instead of printing
```

Comment:
is there a reason why you are not useing shuffleboard

Reply:
I'm not sure how to use shuffleboard. If it's similar to the logging in Vision, I don't know where it goes so I can read it.